### PR TITLE
Changelog for update to 1.3.1 OTel SDK release

### PR DESCRIPTION
**Pull request by Kielek**
Follow up to #631

## Changes

Added info about update of OTel SDK.
Minor fixes.

* [x] Appropriate `CHANGELOG.md` updated for non-trivial changes

### CODE REVIEW

**Review by cijothomas — APPROVED** [review]

File: src/OpenTelemetry.Instrumentation.Hangfire/CHANGELOG.md [hunk 1]
```diff
@@ -1,20 +1,26 @@
 # Changelog
 
+* Update OTel SDK version to `1.3.1`.
```
Comment (utpilla) [inline src/OpenTelemetry.Instrumentation.Hangfire/CHANGELOG.md:3]: This project uses API and not SDK.

File: src/OpenTelemetry.Instrumentation.MySqlData/CHANGELOG.md [hunk 2]
```diff
@@ -2,8 +2,8 @@
 
 ## Unreleased
 
-* Updated OTel SDK package version to 1.3.0
-  ([#508](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/508))
+* Update OTel SDK version to `1.3.1`.
```
Comment (utpilla) [inline src/OpenTelemetry.Instrumentation.MySqlData/CHANGELOG.md:5]: This project uses API and not SDK.

File: src/OpenTelemetry.Instrumentation.Runtime/CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Update OTel SDK version to `1.3.1`.
```
Comment (utpilla) [inline src/OpenTelemetry.Instrumentation.Runtime/CHANGELOG.md:5]: This project uses API and not SDK.

**Comment by utpilla** [comment]
@Kielek Thanks for sending out the PR! Left some comments.

**Pull request merged**